google-account-plugin/src/com/google/cloud/tools/intellij/login/GoogleLoginPrefs.java

```diff
@@ -82,17 +86,23 @@ public static void saveOAuthData(OAuthData credentials) {
    * an {@code OAuthData} object all of whose getters return {@code null} .
    */
   public static OAuthData loadOAuthData() {
-    Preferences prefs = getPrefs();
+    String refreshToken = null;
+    String storedEmail = null;
+    SortedSet<String> storedScopes = new TreeSet<String>();
 
-    String refreshToken = prefs.get(getCustomUserKey(OAUTH_DATA_REFRESH_TOKEN_KEY), null);
-    String storedEmail = prefs.get(getCustomUserKey(OAUTH_DATA_EMAIL_KEY), null);
-    String storedScopesString = prefs.get(getCustomUserKey(OAUTH_SCOPES_KEY), "");
+    if (Services.getLoginService().getActiveUser() != null) {
```

Comment:
one of the big problems in the design of this library is the two sources of state of active user.